src/coreComponents/physicsSolvers/fluidFlow/wells/SinglePhaseWellKernels.cpp

```diff
@@ -573,7 +573,7 @@ PresInitializationKernel::
   real64 const targetBHP = wellControls.getTargetBHP( currentTime );
   real64 const refWellElemGravCoef = wellControls.getReferenceGravityCoef();
   WellControls::Control const currentControl = wellControls.getControl();
-  WellControls::Type const wellType = wellControls.getType();
+  bool const isProducer = wellControls.isProducer();
```

Comment:
Personally I would have called `wellControls.isProducer()` twice instead of storing a variable. This would improve readability (imho).
Also I would have replaced `currentControl == WellControls::Control::BHP` with `wellControls.isBHP()`. But it's pure personal opinion/style.

Reply:
Done